docs/proposals/InputColor/README.md

```diff
@@ -0,0 +1,66 @@
+# InputColor
+
+The `<InputColor>` should allow a user to choose a color. They can either pick a
+color from the color picker or enter a value in the input. When either is
+changed the other should also reflect that change.
+
+## Design Patterns
+
+\_{The goal of the `InputColor` component is to allow a color be chosen to
+reflect the brand of an SP
+
+An SP can either pick a color from the color pick box or enter in the hex value
+and both the color box and the hew value should reflect eachother }\_
+
+## Wireframe
+
+![InputColorWireframe](https://user-images.githubusercontent.com/51097786/62896803-0191c400-bd0f-11e9-9c12-96fbd1c32696.png)
+
+## Interface
+
+```jsx
+render() {
+return (
+      <InputColor
+    placeholder="Button Color"
+    value="#aeab2e"
+    onChange ={onChange}
+  />
+  );
+}
+```
+
+```jsx
+render() {
+return (
+      <InputColor
+    placeholder="Button Color"
+    value="#aeab2e"
+    onChange ={onChange}
+    list="dataListId"
+  />
+  <datalist id={dataListID}>
```

Comment:
I don't follow what the `list` prop and this `datalist` is here?

Reply:
the list binds the datalist to the input. It's used to give the user some options of colors we decide as well as an option to pick their own from the picker(its being used on the component right now)

Reply:
Is DataList another component?

Is it used internally in this one?

Reply:
I don't see it being another component at the moment. It is used with this component. Do you want me to expand on it in the proposal itself?

Reply:
We shouldn't be referencing elements by ID in React like you would in jQuery style code.

If the data list is meant to be an autocomplete menu for the color picker then it should probably be internal to the component.

If that's the case we may want to build an `autocomplete` component and then use that in the color picker here…?